assets/js/index.js

```diff
@@ -154,6 +155,16 @@ $(document).ready(function () {
     }
   });
 
+  $("body").click(function(event) {
+    let sidebar = $(".sidebar") ;
+    if( $(window).width() <= 640 && event.target.className.split(" ",1)!="sidebar" && event.target.className.split(" ",1)!="fas" && sidebar.hasClass("animate__slideInLeft") && (!sidebar.hasClass("hide-sidebar"))){
```

Comment:
There's still this change you haven't made regarding `event.target`. Here's how it should look like:
```
var targetElement = $(event.target);
```
This should be before the if.  

Then use it in the if like this: `!targetElement.hasClass("sidebar")`

Reply:
i tried this before  before ....but it is throwing an error ```split() is not defined```

Reply:
We don't need split anymore in this case. Instead of  `event.target.className.split(" ",1)!="sidebar"` it will just be `!targetElement.hasClass("sidebar")`

Reply:
> We don't need split anymore in this case. Instead of `event.target.className.split(" ",1)!="sidebar"` it will just be `!targetElement.hasClass("sidebar")`

I'll surely do that but i doubt it may cause error anytime...